src/core/types.ts

```diff
@@ -77,6 +78,13 @@ export type SpeakingStateMessage = {
   type: MessageType.State;
 };
 
+export type TranscriberStatusMessage = {
+  data: {
+    status: string;
```

Comment:
thought: still waiting on a reply to my comment [here](https://github.com/agencyenterprise/alpha-ai-avatar-api/pull/174#discussion_r1722357713)

Reply:
I will try to change to a enum in the API

Reply:
Fixed here: https://github.com/agencyenterprise/alpha-ai-avatar-sdk-js/pull/14/commits/dcf054cd9b5e866001f72fa0987b38b68c068ff9